hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NamenodeFsck.java

```diff
@@ -1201,7 +1201,9 @@ private void lostFoundInit(DFSClient dfs) {
         lfInitedOk = true;
       }
     }  catch (Exception e) {
-      e.printStackTrace();
+      if (!lfInitedOk) {
+        throw new IOException("failed to initialize " + lfName);
```

Comment:
This should not lose the original abnormal information。

Reply:
ok, It seems better not use catch.